src/components/structural/header/MyrTour.js

```diff
@@ -55,25 +55,49 @@ class MyrTour extends Component {
     }
 }
 
+const lineBreakHelper = (string) => {
+    return (
+        <div>
+            {string.split("\n").map( (i, key) => {
+                return <div key={key}>{i}</div>;
+            })}
+        </div>
+    );
+};
+
+const view = "The View is where you can see your work. " +
+    "You can navigate the scene with the following controls: \n" +
+    "W: Forward \n" +
+    "A: Left\n" +
+    "S: Backward\n" +
+    "D: Right\n" +
+    "Space: Fly Upwards\n" +
+    "Shift: Fly Downwards\n" +
+    "To view the scene in VR, click the goggles!";
+
+const editor = "This is the editor. You can create 3D scenes using JavaScript " +
+    "and a special set of instructions or functions to MYR.\n The editor can be " +
+    "toggled on and off by opening the settings menu in the top right and " +
+    "clicking the \"View Editor\" switch.";
+
+const stop = "The Stop button will stop the scene. \n Use this to save battery.";
+
 const steps = [
     {
         selector: "#ace-editor",
-        content: "This is the editor. You can create 3D scenes using JavaScript " +
-            "and a special set of instructions or functions to MYR.\n The editor can be " +
-            "toggled on and off by opening the settings menu in the top right and " +
-            "clicking the \"View Editor\" switch."
+        content: lineBreakHelper(editor)
     },
     {
         selector: "#play-btn",
         content: "The Play button will render the scene."
     },
     {
         selector: "#stop-btn",
-        content: "The Stop button will stop the scene. \n Use this to save battery."
+        content: lineBreakHelper(stop)
     },
     {
         selector: "#scene",
-        content: "The View is where you can see your work. \n Click the goggle to view in VR."
+        content: lineBreakHelper(view)
     },
     {
         selector: "#new-btn",
```

Comment:
Can you think of a way to improve the readability of this? I'm not the biggest fan of having two different standards for where the content is stored. Perhaps it would make sense to create a new JSON object with all of the data, and then generate steps by stepping through that object.

Reply:
The more I think about it, the more I know. Let's do this how we do with our autocomplete. Make a JSON object array of the steps. Then map each element of the array into the steps array that gets passed into the autocomplete that calls the formatter on every content item. This way there is one variable with all of the data.